src/Nri/Ui/Tabs/V3.elm

```diff
@@ -0,0 +1,368 @@
+
+                    else
+                        Json.Decode.fail "Wrong key code"
+                )
+                Events.keyCode
+        ]
+        [ Html.styled Html.button
+            [ Css.color Nri.Ui.Colors.V1.navy
+            , Css.hover [ Css.textDecoration Css.none ]
+            , Css.focus [ Css.textDecoration Css.none ]
+            , Css.display Css.inlineBlock
+            , Css.padding4 (Css.px 14) (Css.px 20) (Css.px 12) (Css.px 20)
+            , Css.position Css.relative
+            , Css.textDecoration Css.none
+            , Css.property "background" "none"
+            , Css.fontFamily Css.inherit
+            , Css.fontSize Css.inherit
+            , Css.border Css.zero
+            , Css.cursor Css.pointer
+            ]
+            [ Attributes.fromUnstyled <| Accessibility.Role.tab
+            , Attributes.tabindex -1
+            ]
+            [ viewInnerTab tab ]
+        ]
+
+
+{-| Describe a tab that is meant to link to another page
+-}
+type alias TabLink =
+    { label : String
+    , href : Maybe String
+    }
+
+
+{-| Configure a set a tab links
+-}
+type alias LinkConfig msg =
+    { title : Maybe String
+    , tabs : Zipper TabLink
+    , content : Html msg
+    , alignment : Alignment
+    }
+
+
+{-| View a set of tab links
+-}
+links : LinkConfig msg -> Html msg
+links config =
+    Html.div []
+        [ Html.styled Html.nav
+            [ Css.displayFlex
+            , Css.alignItems Css.flexEnd
+            , Css.borderBottom (Css.px 1)
+            , Css.borderBottomStyle Css.solid
+            , Css.borderBottomColor Nri.Ui.Colors.V1.navy
+            , Nri.Ui.Fonts.V1.baseFont
+            ]
+            []
+            [ config.title
+                |> Maybe.map viewTitle
+                |> Maybe.withDefault (Html.text "")
+            , Html.styled Html.ul
+                (stylesTabsAligned config.alignment)
+                []
+                (config.tabs
+                    |> mapWithCurrent (viewTabLink config)
+                    |> List.Zipper.toList
+                )
+            ]
+        , Html.div [] [ config.content ]
+        ]
+
+
+viewTabLink : LinkConfig msg -> Bool -> TabLink -> Html msg
+viewTabLink config isSelected tabLink =
+    Html.styled Html.li
+        (stylesTabSelectable isSelected)
+        [ Attributes.fromUnstyled <| Accessibility.Role.presentation
+        , Attributes.id (tabToId tabLink)
+        ]
+        [ case tabLink.href of
+            Just href ->
+                Html.styled Html.a
+                    [ Css.color Nri.Ui.Colors.V1.navy
+                    , Css.display Css.inlineBlock
+                    , Css.padding4 (Css.px 14) (Css.px 20) (Css.px 12) (Css.px 20)
+                    , Css.textDecoration Css.none
+                    ]
+                    [ Attributes.href href ]
+                    [ Html.text tabLink.label ]
+
+            Nothing ->
+                Html.styled Html.button
```

Comment:
this is the other. we just can't use anchor tags as buttons anymore. have to use the real thing!

Reply:
Love how upgrading to 19 makes our code more correct <3